ssl/ssl_conf.c

```diff
@@ -397,7 +397,8 @@ static int cmd_Options(SSL_CONF_CTX *cctx, const char *value)
         SSL_FLAG_TBL_INV("ExtendedMasterSecret", SSL_OP_NO_EXTENDED_MASTER_SECRET),
         SSL_FLAG_TBL_INV("CANames", SSL_OP_DISABLE_TLSEXT_CA_NAMES),
         SSL_FLAG_TBL("KTLS", SSL_OP_ENABLE_KTLS),
-        SSL_FLAG_TBL_CERT("StrictCertCheck", SSL_CERT_FLAG_TLS_STRICT)
+        SSL_FLAG_TBL_CERT("StrictCertCheck", SSL_CERT_FLAG_TLS_STRICT),
+        SSL_FLAG_TBL("KTLSTxZerocopySendfile", SSL_OP_ENABLE_KTLS_TX_ZEROCOPY_SENDFILE),
```

Comment:
There is no way to unit test it, because it requires special hardware (a NIC that supports kTLS device offload). I mentioned it in the commit message:

> The unit test is not added, because testing this functionality requires specific hardware and a non-local peer.

Reply:
We can at least test that it is possible to set the option and it doesn't break anything if you do. Testing that it actually *does* anything is a lot harder - but I don't think we need to do that.

Reply:
Yeah, testing for TFO required looking at some behavior, but not erroring out, as long as data still passed.